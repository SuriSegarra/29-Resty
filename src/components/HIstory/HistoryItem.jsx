import React from 'react';
import PropTypes from 'prop-types';

const HistoryItem = () => ();

History.propTypes = {};

export default HistoryItem;
