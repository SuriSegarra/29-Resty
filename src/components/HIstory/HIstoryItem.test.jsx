import React from 'react';
import { shallow } from 'enzyme';
import HistoryItem from './HistoryItem';

describe('HistoryItem component', () => {
  it('renders HistoryItem', () => {
    const wrapper = shallow(<HistoryItem 
      url="https://testurl.com" 
      method='GET'/>);
    expect(wrapper).toMatchSnapshot();
  });
});
