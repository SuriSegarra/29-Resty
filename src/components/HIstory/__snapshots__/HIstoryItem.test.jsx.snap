// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HistoryItem component renders HistoryItem 1`] = `
<section>
  <p>
    https://testurl.com
  </p>
  <p>
    GET
  </p>
</section>
`;
